compiler/rustc_trait_selection/src/solve/normalize.rs

```diff
@@ -98,7 +131,7 @@ where
         );
 
         self.fulfill_cx.register_predicate_obligation(infcx, obligation);
-        let errors = self.fulfill_cx.select_all_or_error(infcx);
+        let errors = self.fulfill_cx.select_where_possible(infcx);
```

Comment:
pls keep making sure we don't have any pending obligations after normalizing each alias.